programs/server/Server.cpp

```diff
@@ -528,16 +528,19 @@ static int readNumber(const String & path)
 
 #endif
 
-static void sanityChecks(Server * server)
+static void sanityChecks(Server & server)
```

Comment:
Also fixed the style. We must use references instead of pointers whenever possible.